autorosters/autorosters.py

```diff
@@ -0,0 +1,42 @@
+from mwrogue.esports_client import EsportsClient
+from rivercogutils import utils
+from redbot.core import commands
+from asyncio import TimeoutError
+
+from .autorosters_main import AutoRostersRunner
+
+
+async def is_lol_staff(ctx) -> bool:
+    for role in ctx.message.guild.roles:
+        if role.name == "LoL-Staff":
+            staff_role = role
+            break
+    if staff_role not in ctx.author.roles:
+        raise commands.UserFeedbackCheckFailure("You don't have enough permissions to run this command!")
+    return True
+
+
+class AutoRosters(commands.Cog):
+    """Automatically generates team rosters for Leaguepedia, using scoreboard data"""
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(pass_context=True)
+    @commands.check(is_lol_staff)
+    async def autorosters(self, ctx, *, overview_page):
+        """Generate team rosters for the specified tournament"""
+        def check(msg):
+            return msg.author == ctx.author and msg.channel == ctx.channel
+        await ctx.send("Tabs: (Eg. `LEC 2022`)")
+        try:
+            tabs = await self.bot.wait_for("message", check=check, timeout=60)
```

Comment:
like I said in discord, I think it's pretty safe to get the tabs from the overview page. In case someone put a navbox or a comment there, you can raise an error "Uhoh the overview page doesn't start with tabs" (that's a wiki content error anyway that should be addressed)